README.md

```diff
@@ -1,2 +1,77 @@
-# rfcs
-RFCs for changes to Google Cartographer projects
+# Cartographer RFCs
+[Cartographer RFCs]: #cartographer-rfcs
+
+For changes more substantial than simple bugfixes and documentation improvements
```

Comment:
one sentence per line.

Reply:
Done

Reply:
no?

Reply:
Oh this is the README.md. I agree that we want ospl in the RFC template to set an example for future RFC authors. I don't see the benefit for the README.md. Actually I would advise against this practice for the README.md as 
- README.md does not change on a regular basis. Inspection of diffs is therefore less frequent
- Using ospl means you have to use a md formatting viewer to properly enjoy the file.